src/main/java/Duke.java

```diff
@@ -6,5 +11,44 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        String input;
+        ArrayList<String> items = new ArrayList<String>();
+        ArrayList<Boolean> marked = new ArrayList<Boolean>();
```

Comment:
Perhaps you could use the array variable name `isMarked` so it sounds like a boolean?